### PR TITLE
[TECH REVIEW] Add basic group support to users

### PR DESCRIPTION
**Pull request by tpoitras**
Ping @liggitt for Tech Review, please.

Original Dev PR: https://github.com/openshift/openshift-docs/pull/480
Docs Trello card: https://trello.com/c/BLbTMsuS
Dev Trello card: https://trello.com/c/pvMpFgIz

### CODE REVIEW

**Comment by liggitt** [comment]
changes look fine, but I don't see this adding any info related to group support

File: architecture/additional_concepts/other_api_objects.adoc [hunk 1]
```diff
@@ -249,18 +249,14 @@ this token, if any.
...
-- If the `*Identity*` already exists, but is not mapped to a `*User*`, login
-fails.
-- If the `*Identity*` already exists, and is mapped to a `*User*`, the user is
-given an `*OAuthAccessToken*` for the mapped `*User*`.
-- If the `*Identity*` does not exist, an `*Identity*`, `*User*`, and
-`*UserIdentityMapping*` are created, and the user is given an
+* If the `*Identity*` already exists, but is not mapped to a `*User*`, then login fails.
+* If the `*Identity*` already exists, and is mapped to a `*User*`, then OpenShift gives the user an `*OAuthAccessToken*` for the mapped `*User*`.
+* If the `*Identity*` does not exist, then OpenShift creates an `*Identity*`, `*User*`, and `*UserIdentityMapping*`. OpenShift also gives the user an
 `*OAuthAccessToken*` for the mapped `*User*`.
```
Comment (adellape) [inline architecture/additional_concepts/other_api_objects.adoc]: FWIW, I've been sticking with `-`s for lists, as I've seen `*`s get funky in certain situations with other `*`s on the same line (not that it seems to affect this particular example).

**Comment by tpoitras** [comment]
Thanks @liggitt & @adellape -- apologies on the delay.

Jordan, I added some context (see line 301), does that sort it out? or is there more to explain? Thanks! :seedling:

**Comment by liggitt** [comment]
Seems ok, though @deads2k's PR for #869 is updating group docs since those are first-class objects now

**Comment by adellape** [comment]
@tpoitras https://github.com/openshift/openshift-docs/pull/869 merged, so this needs a rebase. I added it to https://trello.com/c/1ndBzTQY for minor follow-up, if you want to grab it there and work in your edits from this PR.

**Comment by liggitt** [comment]
We're also deemphasizing the groups field in the User object now… that'll likely get deprecated and go away at some point.

**Comment by tpoitras** [comment]
@liggitt -- if it is being de-emphasized, then should the docs explicitly point it out? Or should I scale this back to the bare minimum?

**Comment by tpoitras** [comment]
Thanks @adellape :ear_of_rice:

**Comment by liggitt** [comment]
scale it back... https://github.com/openshift/openshift-docs/pull/869/files#diff-70e17856cc7f62bd82c0e8880aa2e1f6R369 is what we really want people doing

**Comment by tpoitras** [comment]
PR869 has made this irrelevant. Closing this PR in favour of PR869.

**Pull request closed without merging**